src/cards/vortex.ts

```diff
@@ -1,17 +1,18 @@
 import { getCurrentTargets, Spell } from './index';
 import { CardCategory } from '../types/commonTypes';
 import { playDefaultSpellSFX } from './cardUtils';
-import { pull } from './pull';
+import { pull, pullId } from './pull';
 import { CardRarity, probabilityMap } from '../types/commonTypes';
 
 export const id = 'vortex';
 const spell: Spell = {
   card: {
     id,
+    requires: [pullId],
     category: CardCategory.Movement,
     supportQuantity: true,
     sfx: 'pull',
-    manaCost: 10,
+    manaCost: 20,
```

Comment:
Same as my comment for repel, I think 15 would keep it independent from pull but have it remain cheap so it can be used strategically as a movement spell.